modules/end-user-guide/examples/snip_che-editor-key-values.adoc

```diff
@@ -3,33 +3,6 @@
 |===
 | IDE | `__<editor_key>__` value | Note
 
-
-| link:https://github.com/eclipse-che/che-theia[Che-Theia]
-| `eclipse/che-theia/latest`
-| This is the default IDE that loads in a new workspace without the URL parameter.
-
-| link:https://github.com/eclipse-che/che-theia[Che-Theia]
-| `eclipse/che-theia/next`
-| When using the `next` link:https://github.com/che-incubator/chectl/[chectl] channel, this IDE loads as a default without the URL parameter.
-
-| link:https://github.com/che-incubator/che-code[Visual Studio Code - Open Source]
-| `che-incubator/che-code/insiders`
-| Visual Studio Code Insiders version packaged to run on Kubernetes
-
-| link:https://www.jetbrains.com/help/idea/discover-intellij-idea.html[IntelliJ IDEA]
-| `che-incubator/che-idea/latest`
-| Community Edition - stable version
-
-| link:https://www.jetbrains.com/help/idea/discover-intellij-idea.html[IntelliJ IDEA]
-| `che-incubator/che-idea/next`
-| Community Edition - development version
-
-| link:https://www.jetbrains.com/help/pycharm/quick-start-guide.html[PyCharm]
-| `che-incubator/che-pycharm/latest`
-| Community Edition - stable version
-
-| link:https://www.jetbrains.com/help/pycharm/quick-start-guide.html[PyCharm]
-| `che-incubator/che-pycharm/next`
-| Community Edition - development version
+include::example$snip_{project-context}-list-of-supported-editors.adoc[]
```

Comment:
Why does this file (`modules/end-user-guide/examples/snip_che-editor-key-values.adoc`) exist? It doesn't seem to be included anywhere. And why does it include as non-existent file (`snip_{project-context}-list-of-supported-editors.adoc`)?

Reply:
Thank you for spotting it. Its existence is a mistake. I'm removing it.

Reply:
First, i replaced it with modules/end-user-guide/examples/snip_che-table-of-supported-editors.adoc. Then, i migrated the remaining lines into the procedure modules.

Reply:
My mistake. I initially migrated most of its content into another file, and then migrated the remaining lines into the procedure modules.
Removing this file.
Thank you for spotting it!